plugins/chat/assets/javascripts/discourse/models/chat-message.js

```diff
@@ -13,9 +15,8 @@ export default class ChatMessage {
     return new ChatMessage(channel, args);
   }
 
-  static createStagedMessage(channel, args = {}) {
-    args.id = guid();
-    args.staged = true;
+  static createDraftMessage(channel, args = {}) {
+    args.draft = true;
     return new ChatMessage(channel, args);
```

Comment:
Small nitpick, but you could just call `return create(channel, args)` here right?

Reply:
`return ChatMessage.create(channel, args);` yes

Reply:
Oh @jjaffeux I meant use the `static create` function just above this one :)

Reply:
Ah OK all good